google/cloud/storage/bucket_metadata.h

```diff
@@ -212,6 +254,7 @@ class BucketMetadata : private internal::CommonMetadata<BucketMetadata> {
   BucketBilling billing_;
   std::vector<CorsEntry> cors_;
   std::vector<ObjectAccessControl> default_acl_;
+  BucketEncryption encryption_;
```

Comment:
Note: At some point, we should consider whether non-required fields should be wrapped with `optional<>`. That refactoring probably extends beyond just `encryption` and belongs in a separate commit, so I've I filed #934 for this.

Reply:
Ack. Thanks for creating the bug btw.